examples/cms-wordpress/vercel.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I believe we want to move away from a `vercel.json` and into the environment UI. We should consult with Leo on this.

Reply:
I agree, we should update other examples as well.

Reply:
We'll do this once the new import flow is out